### ISSUE TITLE
Fix font sizes

### ISSUE THREAD

**Issue by WesleyBranton** [WesleyBranton/New-Browser-Style#14]
Font sizes are being set by the CSS instead of using relative font sizes based on the browser settings. Also, the font size is being set in pixels for some areas of the library, which is a poor practice and fails the success criteria for 1.4.4 of WCAG AA.

See #3.
